src/internal/connector/onedrive/item.go

```diff
@@ -23,9 +23,14 @@ var downloadURLKeys = []string{
 	"@content.downloadUrl",
 }
 
+type augmenterAndGetter interface {
```

Comment:
Do we need this? Can we just pass `itemGetterAndAugmenter` to `downloadItem`?

Reply:
fixed in later PR